### ISSUE TITLE
Java error 5 (access denied) when running install-apks

### ISSUE THREAD

**Issue by tidanneels** [google/bundletool#158]
**Describe the bug**
When I try to run the `install-apks` command I have a Java error : `Cannot run program "C:\adb": CreateProcess error=5, Access denied`.

**Bundletool version(s) affected**
Version: 0.13.4

**Stacktrace**
```PS C:\src\projects\simuaube\build\app\outputs\bundle\release> java -jar bundletool.jar install-apks --apks=app-release.apks
04:03:04 E/adb: Cannot run program "C:\adb": CreateProcess error=5, Accès refusé
[BT:0.13.4] Error: java.io.IOException: Cannot run program "C:\adb": CreateProcess error=5, Accès refusé
java.lang.IllegalArgumentException: java.io.IOException: Cannot run program "C:\adb": CreateProcess error=5, Accès refusé
        at com.android.ddmlib.AndroidDebugBridge.<init>(AndroidDebugBridge.java:602)
        at com.android.ddmlib.AndroidDebugBridge.createBridge(AndroidDebugBridge.java:380)
        at com.android.tools.build.bundletool.device.DdmlibAdbServer.init(DdmlibAdbServer.java:78)
        at com.android.tools.build.bundletool.commands.InstallApksCommand.execute(InstallApksCommand.java:160)
        at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:88)
        at com.android.tools.build.bundletool.BundleToolMain.main(BundleToolMain.java:46)
Caused by: java.io.IOException: Cannot run program "C:\adb": CreateProcess error=5, Accès refusé
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
        at com.android.ddmlib.AndroidDebugBridge$1.run(AndroidDebugBridge.java:660)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: CreateProcess error=5, Accès refusé
        at java.lang.ProcessImpl.create(Native Method)
        at java.lang.ProcessImpl.<init>(ProcessImpl.java:444)
        at java.lang.ProcessImpl.start(ProcessImpl.java:139)
        at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
        ... 2 more
```

**To Reproduce**
Run this command : `bundletool install-apks --apks=file.apks` with file.apks in the current directory and an android device connected to your computer

**Expected behavior**
It should install the apk to my device

**Environment:**
Windows 10, Android 10, Powershell run as administrator

**Additional context**
 I think it is because it runs "C:\adb" and not "C:\adb\adb.exe" but I don't know how to set a custom adb path

**Comment by plecesne** [comment]
Thank you for the report.

That looks like a bug indeed. While waiting for it to be fixed, you can add the flag `--adb C:\adb\adb.exe` to the command.

You can also see other flags available in commands by using: `bundletool help <command>`.

**Comment by ymakhno** [comment]
Fixed in 0.15.0.
